caffe2/opt/onnxifi_transformer.cc

```diff
@@ -142,6 +142,14 @@ OnnxifiTransformer::OnnxifiTransformer(bool infer_shapes, bool debug)
       ONNXIFI_STATUS_SUCCESS);
 }
 
+OnnxifiTransformer::~OnnxifiTransformer() {
+  for (unsigned i = 0; i < num_backends_; ++i) {
+    if (lib_->onnxReleaseBackendID(backend_ids_[i]) != ONNXIFI_STATUS_SUCCESS) {
```

Comment:
nit: why don't we just iterate over backend_ids_? what's the purpose of num_backends_?

Reply:
No special reason. I got caught up in C-style coding in onnxifi. :)